third_party/dummy-hook/CMakeLists.txt

```diff
@@ -4,8 +4,12 @@ if(GIT_FOUND)
       WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
       OUTPUT_VARIABLE Ginkgo_GIT_HOOKS_DIR
       OUTPUT_STRIP_TRAILING_WHITESPACE)
+    # If it is not in git, give an error and return
+    if(NOT Ginkgo_GIT_HOOKS_DIR)
+        message(WARNING "Not in a git repository")
```

Comment:
Can you add some more context to the message (`, no git hook was installed`), and maybe replace it by a `STATUS`? Otherwise users get this large, visible warning for something that only matters for developers.